deploy/deployment/openshift/org_v2_checluster.yaml

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright (c) 2019-2021 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
+
+apiVersion: org.eclipse.che/v2
+kind: CheCluster
+metadata:
+  name: eclipse-che
+  namespace: eclipse-che
+spec:
+  workspaces:
+    defaultNamespace:
+      template: '<username>-che'
+    storage:
+      pvcStrategy: 'common'
+      preCreateSubPaths: true
```

Comment:
@amisevsk @l0rd Wondering if we still need `preCreateSubPaths` property? as I recall it was needed only for the old version of k8s / openshift

Reply:
By default, DWO has logic to work around the same issue to this day. However, now that PVC provisioning is managed by DWO, perhaps this option is no longer relevant to the CheCluster CR.

Reply:
Thank you. I will remove it then.

Reply:
But what about `PvcJobsImage` then? Does somebody use it?
Description says:
```
	// Overrides the container image used to create sub-paths in the Persistent Volumes.
	// This includes the image tag. Omit it or leave it empty to use the default container image provided by the Operator.
```

Reply:
I think it makes sense to drop as well -- DWO handles PVC jobs and has its own related image for this, so this field will do nothing currently. If one day there's a mechanism within Che to configure DWO, then it might be useful then, but we can always add it at that point.

For the CheCluster API, I would err on the side of fewer fields, as adding fields is backwards compatible but removing them is not.